source/ios/tools/AdaptiveCards.podspec

```diff
@@ -17,6 +17,7 @@ Pod::Spec.new do |spec|
 
   spec.subspec 'AdaptiveCardsCore' do | sspec |
     sspec.source_files = 'source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/*.{h,m,mm}'
+    sspec.resource_bundles = {'AdaptiveCards' => ['source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/Resources/**/*']}
```

Comment:
Does this need to stay as part of the full podspec as well?

Reply:
It should not be necessary. AdaptiveCardsCore is a default subspec, the way the SDK is consumed pre 2.8.3 is by:
'AdaptiveCards' ~> 'x.x.x'  . In 2.8.3 consuming it the same way will automatically pull AdaptiveCardsCore.